language-plutus-core/test/Generators.hs

```diff
@@ -30,13 +30,7 @@ genKind = simpleRecursive nonRecursive recursive
           recursive = [KindArrow emptyPosn <$> genKind <*> genKind]
 
 genBuiltinName :: MonadGen m => m BuiltinName
-genBuiltinName = Gen.choice $ pure <$>
-    [ AddInteger, SubtractInteger, MultiplyInteger, DivideInteger, RemainderInteger
-    , LessThanInteger, LessThanEqInteger, GreaterThanInteger, GreaterThanEqInteger
-    , EqInteger, ResizeInteger, IntToByteString, Concatenate, TakeByteString
-    , DropByteString, ResizeByteString, SHA2, SHA3, VerifySignature
-    , EqByteString, TxHash, BlockNum, BlockTime
-    ]
+genBuiltinName = Gen.element allBuiltinNames
```

Comment:
I made the list of all built-in names. This way we need to update less places if we add a new built-in. And unlike other places, here we wouldn't get a warning.